File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        n = len(details)
        n_char = [len(s) for s in details]
        count = 0
        for i in range(n):
            if int(details[i][11]) * 10 + int(details[i][12]) > 60:
                count += 1
        return count
                
            

File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.cpp
using namespace std;

class Solution {
public:
    int countSeniors(vector<string>& details) {
        int n = details.size();
        int count = 0;
        for(int i = 0; i<n; ++i){
            int age = int(details[i][11] - '0') * 10 + int(details[i][12] - '0');
                if(age > 60){
                    count += 1;
                }
                
            }
        
                     
    return count;
    }
};